base/src/main/res/values/dimens.xml

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <integer name="player_subtitles_default_text_size">16</integer>
```

Comment:
It is not used any more. Before this values was in 2 places but now it is just in one place (PreferencesHandler)